assets/src/edit-story/components/popup/index.js

```diff
@@ -72,16 +74,21 @@ const Container = styled.div.attrs(
   /*
    * Custom gray scrollbars for Chromium & Firefox.
    */
-  * {
-    scrollbar-width: thin;
-    scrollbar-color: ${({ theme }) => theme.colors.bg.v10}
-      ${({ theme }) => theme.colors.bg.v12};
-  }
 
-  *::-webkit-scrollbar {
-    width: ${SCROLLBAR_WIDTH}px;
-    height: ${SCROLLBAR_WIDTH}px;
-  }
+  ${({ obtrusiveScrollbars }) =>
```

Comment:
@barklund / @merapi Do you know if there's a way how to trigger scrolling in Karma tests? `scrollTo` scrolls but without the scrollbar, wondering if there's a way how to trigger the scrollbar actually appearing, too.